src/metrics/hpr_metrics.erl

```diff
@@ -52,6 +76,15 @@ handle_cast(_Msg, State) ->
 
 handle_info(?METRICS_TICK, State) ->
     lager:debug("running metrics"),
+    _ = erlang:spawn_opt(
```

Comment:
A minor concern here, is if `record_grpc_connections` get stuck or is slow, we could spawn off a lot of these. Consider some code to make sure there is only one of these running at a time? We had something similar to this at my last company. When the system got bogged down, the metrics collection occasionally took longer than interval between collections which lead to additional load, just when we didn't need it, and confusing metrics.

Reply:
NM - it's not an interval send by send after.

Reply:
Good note actually saw this once.

Reply:
Right - the send after is outside the spawn, so there could be more than one running at a time.